gridfs/grid_file.py

```diff
@@ -152,12 +150,14 @@ def __init__(
           - `session` (optional): a
             :class:`~pymongo.client_session.ClientSession` to use for all
             commands
-          - `disable_md5` (optional): When True, an MD5 checksum will not be
-            computed for the uploaded file. Useful in environments where
-            MD5 cannot be used for regulatory or other reasons. Defaults to
-            False.
           - `**kwargs` (optional): file level options (see above)
 
+        .. versionchanged:: 4.0
+           Removed the `disable_md5` parameter.
```

Comment:
And here.

Reply:
Updated to link to the migration guide which describes the change in more detail.